advanced/other_topics.rst

```diff
@@ -37,9 +64,101 @@ To be completed
 
 
 Puzzles
----------------
+-------
 To be completed
 
 
-Adding a company logo cutscene
-------------------------------
+States verses animations
+------------------------
+The features offered by the state and animation commands are rather similar.
+This documentation section is to help you choose when to use a particular
+command.
+
+``anim`` is used for starting an animation and immediately returning control
+to the next command in the script. This is good for animations that will run
+once (like part of a cutscene), allowing you to have furher script commands
+running while the animation is executing. As an example, you could start an
+animation of a bird flying across the screen, then a separate one of a car
+driving along - both animations would run simultaneously.
+
+``anim_block`` is used when you want to control the pacing of your game /
+cutscene, and ensure that an animation completes before further game commands
+are processed. As an example, if you had separate animations of someone walking
+up to a door and another showing them opening it, using ``anim_block`` to play
+the person walking animation would ensure they reached the door before
+it was opened.
+
+``set_state`` is like combining a "set global" command with an "anim" command.
+It retains a state which can be queried (e.g. checking if the door is in an
+unlocked or a locked state) and runs an animation (if an animation with the
+same name exists in this room). State is preserved between rooms, meaning that
+if you set door 2's state to "dooropen" and leave the room, when you come back,
+if you query the state of door 2, it will still return "dooropen".
+
+
+
+** Comment required here about whether state will play with "immediate" or not
+when you reenter a room - still under discussion
+
+** Comment required here once ESC script supports querying the current state
+of an ESCItem
+
+
+
+.. _logo-label:
+
+Adding a company logo or introductory cutscene
+----------------------------------------------
+A company logo or introductory cutscene that plays before the menu of your
+game is displayed is optional for your game. To create one:
+
+* Create an ``ESCRoom`` to display your logo or cutscene introduction.
+  Depending on your requirements this may be any combination of Escoria
+  commands and animations driven by Godot ``AnimationPlayers``.
+
+* Create an Escoria script (a text file with a ".esc" extension) and attach it
```

Comment:
```suggestion
* Create an ESC script and attach it
```

Reply:
I think, "ESC script" should be a known term for the readers by the time they reach this documentation.